CMakeLists.txt

```diff
@@ -434,26 +450,6 @@ configure_package_config_file(${PROJECT_NAME}Config.cmake.in
                               "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
                               INSTALL_DESTINATION ${PROJECT_BINARY_DIR})
 
-# Install xeus-cpp tag files
-set(XEUS_CPP_DATA_DIR "share/xeus-cpp" CACHE STRING "xeus-cpp data directory")
-set(XCPP_TAGFILES_DIR ${CMAKE_CURRENT_SOURCE_DIR}/share/xeus-cpp/tagfiles)
-install(DIRECTORY ${XCPP_TAGFILES_DIR}
-        DESTINATION ${XEUS_CPP_DATA_DIR})
-
-set(XEUS_CPP_CONF_DIR "etc/xeus-cpp"   CACHE STRING "xeus-cpp configuration directory")
-set(XCPP_TAGCONFS_DIR ${CMAKE_CURRENT_SOURCE_DIR}/etc/xeus-cpp/tags.d)
-install(DIRECTORY ${XCPP_TAGCONFS_DIR}
-        DESTINATION ${XEUS_CPP_CONF_DIR})
-
-# Add definitions for the kernel to find tagfiles.
-add_definitions(-DXCPP_TAGFILES_DIR="${CMAKE_INSTALL_PREFIX}/${XEUS_CPP_DATA_DIR}/tagfiles")
-if(CMAKE_INSTALL_PREFIX STREQUAL "/usr")
-    # install into /etc instead of /usr/etc
-    add_definitions(-DXCPP_TAGCONFS_DIR="/${XEUS_CPP_CONF_DIR}/tags.d")
-else()
-    add_definitions(-DXCPP_TAGCONFS_DIR="${CMAKE_INSTALL_PREFIX}/${XEUS_CPP_CONF_DIR}/tags.d")
-endif()
```

Comment:
Why removing this? It is required for regular unix-like system-wide installs.

Reply:
Rearranged it according to the code structure. Didn't remove per say.